Changes/Modifications to the import/export guide (#554)
theforeman/foreman-documentation: guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

You can use the archive that the `hammer content-export` command outputs to create a version of a Content View with the same content as the exported Content View version.
For more information about exporting a Content View version, see xref:Using_ISS-Exporting-a-Content-View-Version[].

> When you import a Content View version, it has the same major and minor version numbers and contains the same repositories with the same packages and errata. The Custom Repositories, Products and Content Views are automatically created if they don't exist in the importing organization.

Reviewer: ```suggestion
When you import a Content View version, it has the same major and minor version numbers and contains the same repositories with the same packages and errata. 
The Custom Repositories, Products and Content Views are automatically created if they do not exist in the importing organization.
```

> When you import a Content View version, it has the same major and minor version numbers and contains the same repositories with the same packages and errata. The Custom Repositories, Products and Content Views are automatically created if they don't exist in the importing organization.

Reviewer 2: ```suggestion
When you import a Content View version, it has the same major and minor version numbers and contains the same repositories with the same packages and errata.
The Custom Repositories, Products and Content Views are automatically created if they don't exist in the importing organization.
```

.Prerequisites

To import a Content View, ensure that {ProjectServer} where you want to import meets the following conditions:

> . The exported dump must be in a directory under `/var/lib/pulp/imports` . More over the directory must have `pulp:pulp` permissions for pulp to be able to read and write the json files in that directory.

Reviewer: ```suggestion
. The exported dump must be in a directory under `/var/lib/pulp/imports`. 
. The directory must have `pulp:pulp` permissions so that Pulp can read and write the `.json` files in that directory.
```

> . The exported dump must be in a directory under `/var/lib/pulp/imports` . More over the directory must have `pulp:pulp` permissions for pulp to be able to read and write the json files in that directory.

Reviewer 2: ```suggestion
. The exported dump must be in a directory under `/var/lib/pulp/imports`.
More over the directory must have `pulp:pulp` permissions for pulp to be able to read and write the json files in that directory.
```

Is `archive` a correct alternative to `dump`?

Author: changed to archive :)

.Export with chunking

> In many cases the exported archive content may be several gigabytes in size. We may want to split it smaller sizes or chunks. We can use the `--chunk-size-gb` flag directly in the export command to handle this. In the example below we specify `--chunk-size-gb=2` to split the archives in `2 GB` chunks.

Reviewer: ```suggestion
In many cases the exported archive content may be several gigabytes in size. 
If you want to split it into smaller sizes or chunks. 
You can use the `--chunk-size-gb` flag directly in the export command to handle this. 
In the following example, you can see how to specify `--chunk-size-gb=2` to split the archives in `2 GB` chunks.
```

|Export a content view version promoted  to the Dev Environment|`hammer content-export complete version --content-view=view --organization="Default_Organization" --lifecycle-environment=’Dev’`

> |Export a content view in smaller chunks (200 mb slabs)|`hammer content-export complete version --content-view=view --version=1.0 --organization="Default_Organization" --chunk-size-gb=2`

Reviewer 2: Does `200 mb slabs` also have to be changed?

Author: thanks!
